### DISCUSSION TITLE
AP without wifi connection possible?

### DISCUSSION THREAD

**Discussion by billie-sandbox** [RaspAP/raspap-webgui#1565] [Q&A]
Pi : raspberry pi 5
Os: headless raspberry os lite
Additional info: i install nextcloud in docker

I need to test something raspap, which is accessing nextcloud through device that connected via raspap to my pi, without pi being connected to network.

Issue :
-  i dont know and can not find how to only disconnect or forget wifi for my pi (from cli because i use headless os, and from raspap), all i find is how to turn off wifi (which turn off the wlan0 too, as far as i know, the AP rely on this interface too)
- tried deleting network data on wpa suplicant config, still connected
- and then, turning off my router. Then i check from my ipad and laptop, the AP network is gone too.

Is AP without a connected network is not possible?

**Comment by billz** [comment]
Not sure I understand your question. Are you asking if RaspAP can host an AP without an internet connection?
